src/rust/rust-toolchain

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.51.0"
+channel = "nightly"
```

Comment:
We don't wanna do this.

Reply:
Ahh yea I forgot to revert this. I had to fix a bug that was caused by this file actually